packages/itwinui-css/src/status-message/status-message.scss

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@use '../mixins';
+
+.iui-status-message-content {
+  font-size: var(--iui-font-size-0);
+  color: var(--iui-color-text-muted);
+}
+
+.iui-status-message-icon {
```

Comment:
is this whole class even needed? we can just set gap on the status-message and use `--_iui-svg-fill` for the status fill

Reply:
Updated statusmessage